src/routes/components/components.json

```diff
@@ -2705,5 +2705,151 @@
 		"addedOn": "2023-08-14",
 		"category": "Data Visualisation",
 		"tags": ["components and libraries", "images"]
+	},
+	{
+		"title": "svelte-transition",
+		"url": "https://captaincodeman.github.io/svelte-headlessui/transition/",
+		"repository": "https://github.com/captaincodeman/svelte-transition",
+		"description": "Svelte component to apply CSS classes matching the syntax in TailwindUI components",
+		"npm": "svelte-transition",
+		"addedOn": "2023-08-16",
+		"category": "User Interaction",
+		"tags": ["components and libraries", "animations", "typescript"]
+	},
+	{
+		"title": "svelte-headlessui",
+		"url": "https://captaincodeman.github.io/svelte-headlessui/",
+		"repository": "https://github.com/captaincodeman/svelte-headlessui",
+		"description": "A very lightweight unofficial implementation of Tailwind HeadlessUI",
+		"npm": "svelte-headlessui",
+		"addedOn": "2023-08-16",
+		"category": "Design System",
+		"tags": ["components and libraries", "component sets", "typescript"]
+	},
+	{
+		"title": "svelte-color-select",
+		"url": "https://captaincodeman.github.io/svelte-color-select/",
+		"repository": "https://github.com/captaincodeman/svelte-color-select",
+		"description": "OKHSV Color Selection component for Svelte using OKLab perceptual colorspace",
+		"npm": "svelte-color-select",
+		"addedOn": "2023-08-16",
+		"category": "",
+		"tags": ["components and libraries", "typescript"]
+	},
+	{
+		"title": "svelte-form-helper",
+		"url": "https://captaincodeman.github.io/svelte-form-helper/",
+		"repository": "https://github.com/captaincodeman/svelte-form-helper",
+		"description": "Lightweight helper for form validation with Svelte",
+		"npm": "svelte-form-helper",
+		"addedOn": "2023-08-16",
+		"category": "Forms & User Input",
+		"tags": [
+			"inputs and widgets",
+			"forms and validation",
+			"validation",
+			"typescript",
+			"components and libraries"
+		]
+	},
+	{
+		"title": "svelte-theme-select",
+		"url": "https://captaincodeman.github.io/svelte-theme-select/",
+		"repository": "https://github.com/captaincodeman/svelte-theme-select",
```

Comment:
this one has less than 15 stars, so perhaps we should hold off on it https://github.com/svelte-society/sveltesociety.dev/pull/442